# Timer related statistics
float32 game_time_duration
float32 game_elapsed_time
float32 game_time_left
bool is_active